jupyter/datascience/ubi8-python-3.8/test/test_notebook.ipynb

```diff
@@ -0,0 +1,243 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "01399e50-95a4-4378-9f8d-434681d6bbf4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import unittest\n",
+    "from unittest import mock\n",
+    "from platform import python_version\n",
+    "import pandas as pd\n",
+    "from pandas.testing import assert_frame_equal\n",
+    "import numpy as np\n",
+    "import scipy\n",
+    "import sklearn\n",
+    "from scipy import special\n",
+    "from scipy import integrate\n",
+    "from sklearn import datasets\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "import matplotlib\n",
+    "import matplotlib.pyplot as plt\n",
+    "import kafka\n",
+    "from kafka import KafkaConsumer, KafkaProducer, TopicPartition\n",
+    "from kafka.producer.buffer import SimpleBufferPool\n",
+    "from kafka import KafkaConsumer\n",
+    "from kafka.errors import KafkaConfigurationError\n",
+    "import boto3\n",
+    "import kfp_tekton\n",
+    "import kfp\n",
+    "from kfp import LocalClient, run_pipeline_func_locally\n",
+    "\n",
+    "def get_major_minor(s):\n",
+    "    return '.'.join(s.split('.')[:2])\n",
+    "\n",
+    "class TestPythonVersion(unittest.TestCase):\n",
+    "    def test_version(self):\n",
+    "        expected_major_minor = '3.8'\n",
+    "        actual_major_minor = get_major_minor(python_version())\n",
+    "        self.assertEqual(actual_major_minor, expected_major_minor, \"incorrect version\")\n",
```

Comment:
What about (similar for other such occurrences):

```suggestion
    "        self.assertEqual(actual_major_minor, expected_major_minor, \"incorrect Python version\")\n",
```

But, maybe the failing message is clear so my comment is irrelevant.

Reply:
@rkpattnaik780 looks good, sorry for bugging :slightly_smiling_face: